### DISCUSSION TITLE
Cloudflare Hono with cron event

### DISCUSSION THREAD

**Discussion by Nytrm** [honojs/hono#1087] [Q&A] [answered]
Is it already possible to add a scheduled event for cloudflare workers with hono ?

https://developers.cloudflare.com/workers/runtime-apis/scheduled-event/

**Comment by yusukebe** [comment, marked as answer]
Hi @Nytrm 

You can create a worker with a scheduled event and a Hono app like this:

```ts
const app = new Hono()

//...

export default {
  fetch: app.fetch,
  async scheduled(event, env, ctx) {
    ctx.waitUntil(doSomeTaskOnASchedule())
  },
}
```
